<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TocBar</class>
 <widget class="QDockWidget" name="TocBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>62</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="findEdit">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Find in page</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fNextButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/kiwix.qrc">
        <normaloff>:/icons/search_forward.svg</normaloff>:/icons/search_forward.svg</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fPreviousButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/kiwix.qrc">
        <normaloff>:/icons/search_backward.svg</normaloff>:/icons/search_backward.svg</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="hideButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resources/kiwix.qrc">
        <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/kiwix.qrc"/>
 </resources>
 <connections/>
</ui>
